{"hierarchy":{"paths":[["doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX"]]},"kind":"symbol","sections":[],"schemaVersion":{"minor":3,"major":0,"patch":0},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"NSGetUncaughtExceptionHandler"},{"text":"() -> ((","kind":"text"},{"preciseIdentifier":"c:objc(cs)NSException","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/NSException","kind":"typeIdentifier","text":"NSException"},{"kind":"text","text":") -> "},{"preciseIdentifier":"s:s4Voida","kind":"typeIdentifier","text":"Void"},{"kind":"text","text":")?"}],"platforms":["macOS"],"languages":["swift"]}]}],"metadata":{"title":"NSGetUncaughtExceptionHandler()","modules":[{"name":"FoundationX"}],"externalID":"c:@F@NSGetUncaughtExceptionHandler","role":"symbol","symbolKind":"func","fragments":[{"kind":"keyword","text":"func"},{"text":" ","kind":"text"},{"text":"NSGetUncaughtExceptionHandler","kind":"identifier"},{"text":"() -> ((","kind":"text"},{"text":"NSException","kind":"typeIdentifier","preciseIdentifier":"c:objc(cs)NSException"},{"text":") -> ","kind":"text"},{"text":"Void","preciseIdentifier":"s:s4Voida","kind":"typeIdentifier"},{"text":")?","kind":"text"}],"roleHeading":"Function"},"variants":[{"paths":["\/documentation\/foundationx\/nsgetuncaughtexceptionhandler()"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/NSGetUncaughtExceptionHandler()"},"references":{"doc://com.swallow.documentation.FoundationX/documentation/FoundationX":{"abstract":[],"kind":"symbol","title":"FoundationX","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX","url":"\/documentation\/foundationx","role":"collection","type":"topic"},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/NSException":{"fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"text":"NSException","kind":"identifier"}],"navigatorTitle":[{"text":"NSException","kind":"identifier"}],"kind":"symbol","role":"symbol","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/NSException","url":"\/documentation\/foundationx\/nsexception","type":"topic","abstract":[],"title":"NSException"},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/NSGetUncaughtExceptionHandler()":{"kind":"symbol","title":"NSGetUncaughtExceptionHandler()","type":"topic","role":"symbol","url":"\/documentation\/foundationx\/nsgetuncaughtexceptionhandler()","fragments":[{"kind":"keyword","text":"func"},{"text":" ","kind":"text"},{"text":"NSGetUncaughtExceptionHandler","kind":"identifier"},{"text":"() -> ((","kind":"text"},{"text":"NSException","preciseIdentifier":"c:objc(cs)NSException","kind":"typeIdentifier"},{"text":") -> ","kind":"text"},{"text":"Void","kind":"typeIdentifier","preciseIdentifier":"s:s4Voida"},{"text":")?","kind":"text"}],"abstract":[],"identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/NSGetUncaughtExceptionHandler()"}}}